src/commands/kv/bucket/upload.rs

```diff
@@ -84,15 +84,15 @@ pub fn upload_files(
     Ok(asset_manifest)
 }
 
-fn call_put_bulk_api(
+fn call_api(
     target: &Target,
-    user: GlobalUser,
+    user: &GlobalUser,
     namespace_id: &str,
     key_value_batch: &mut Vec<KeyValuePair>,
 ) -> Result<(), failure::Error> {
     message::info("Uploading...");
```

Comment:
i would argue that `call_api` functions should not use message or any other stdout calls, if at all possible. those should be implemented by the caller.

Reply:
Agree!

Reply:
oh wait, this function shouldn't even be called `call_api`; it doesn't reference cloudflare-rs directly at all. Let me rename this; we should consider this another wrapper function around the true API call.